api/src/constants.js

```diff
@@ -7,3 +7,20 @@ export const RESOURCE_VERBS = {
   UPDATE: 'UPDATE',
   CREATE: 'CREATE',
 };
+
+export const FILE_PATH = 'app-web/topicRegistry/';
+
+export const COMMIT_MESSAGE = {
+  CREATE: 'add new topic',
+  UPDATE: 'edit topic',
+};
+
+export const PR_TITLE = {
+  CREATE: 'Suggest new topic to the devhub',
+  UPDATE: 'Suggest an edit to topic',
+};
+
+export const PR_BODY = {
+  CREATE: 'Add new topic to the devhub',
```

Comment:
what do oyu think of having the pr body create and update have information on the topic.